index.html

```diff
@@ -1319,6 +1320,32 @@ <h3><dfn>close()</dfn> method</h3>
       <li>Return |promise|.
     </ol>
   </section>
+  <section>
+    <h3><dfn>forget()</dfn> method</h3>
+
+    <aside class="example">
+      It is posssible to voluntarily revoke a permission to a serial port that was granted by a user.
+
+      <pre class="js">
+        // Request a serial port.
+        const port = await navigator.serial.requestPort();
+
+        // Then later... revoke permission to the serial port.
+        await port.forget();
+      </pre>
+    </aside>
+
+    The {{SerialPort/forget()}} method steps are:
+
+    <ol>
+      <li>Let |promise| be [=a new promise=].
+      <li>
+        Remove |this| from the sequence of available serial ports on
+        the system which the user has allowed the site to access as the
+        result of a previous call to {{Serial/requestPort()}}.
```

Comment:
I feel like we should specify the effect that this will have. For example, what happens if the port was already open? What happens if `open()` is called after this? Intuitively I think this should have the same effect as if the port was disconnected. That is, it is an ungraceful termination of the site's connection to the device and might even fire a `disconnect` event. It could also be an error to call this method if the port is open.

Reply:
I've added a `"forgotten"` state as in WebHID to track this: https://wicg.github.io/webhid/#forget-method
Thanks for flagging this issue!